src/Compilers/CSharp/Portable/BoundTree/BoundNodes.xml

```diff
@@ -77,7 +77,7 @@
   </AbstractNode>
 
   <AbstractNode Name="BoundValuePlaceholderBase" Base="BoundExpression">
-    <Field Name="Type" Type="TypeSymbol" Override="true" Null="disallow"/>
+    <Field Name="Type" Type="TypeSymbol" Override="true"/>
```

Comment:
I didn't find any place where we take advantage of this change. Revert?

---
In reply to: [68790405](https://github.com/dotnet/roslyn/pull/12224#discussion_r68790405) [](ancestors = 68790405)